RELEASE-NOTES.md

```diff
@@ -0,0 +1,11 @@
+# Release Notes
+
+## Version 3.2.0
+
+### PR 313
```

Comment:
PR 313 went into another development branch not into main. so that can be removed here.